src/utils.py

```diff
@@ -275,10 +276,61 @@ def _get_method(obj, method_path):
     return obj
 
 
-def _ensure_do_folder():
+def _ensure_do_folder() -> None:
     """makes sure that ~/.deeporigin exists"""
 
     deeporigin_path = Path.home() / ".deeporigin"
 
     if not deeporigin_path.exists():
         deeporigin_path.mkdir(parents=True)
+
+
+def _download_nucleus_api_spec():
```

Comment:
this helps us download the spec for later

Reply:
Curious - why do we need it downloaded to disk?

Reply:
solely for me to update the SDK later on!